ECHO-COVID19.ipynb

```diff
@@ -79,10 +83,18 @@
     "import csv\n",
     "import datetime\n",
     "import folium\n",
-    "import ipywidgets as widgets\n",
-    "\n",
+    "import ipywidgets as widgets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "# Set up some default parameters for graphing\n",
     "from matplotlib import cycler\n",
+    "colour = \"#00C2AB\" # The default colour for the barcharts\n",
```

Comment:
just to check, `color` and `colour` both work?

Reply:
`colour` is a variable representing the hex code for the barchart fills. `color` is the name of a parameter for many matplot functions.